README.md

```diff
@@ -142,6 +144,41 @@ convivaAnalytics.updateContentMetadata(metadata);
 
 Those values will be cleaned up after the session is closed.
 
+### Server Side Ad Tracking
+
+In order to track server side ads you can use the methods provided in `SsaiApi` which can be accessed via `ConvivaAnalytics.ssai`.
+The following example shows basic server side ad tracking:
+
+```swift
```

Comment:
```suggestion
```ts
```

Reply:
oops, thanks fixed it by accepting the suggestion!